clap-utils/src/compute_budget.rs

```diff
@@ -43,4 +43,6 @@ pub enum ComputeUnitLimit {
     Static(u32),
     /// Simulate the transaction to find out the compute unit usage
     Simulated,
+    /// Simulate the transaction and add a small percentage to account for potential drift
+    SimulatedWithExtra(u8),
```

Comment:
even though we cast to u64 100% of the time, i made this u8 to make it impossible to chose a value higher than 256

Reply:
[aab16f5](https://github.com/anza-xyz/agave/pull/7447/commits/aab16f5003134b3b2bbdfda4681f82d51d074449)